content/docs/setup/kubernetes/multicluster-install.md

```diff
@@ -192,14 +192,17 @@ cluster to the Istio control plane.
 $ export PILOT_POD_IP=$(kubectl -n istio-system get pod -l istio=pilot -o jsonpath='{.items[0].status.podIP}')
 $ export POLICY_POD_IP=$(kubectl -n istio-system get pod -l istio=mixer -o jsonpath='{.items[0].status.podIP}')
 $ export STATSD_POD_IP=$(kubectl -n istio-system get pod -l istio=statsd-prom-bridge -o jsonpath='{.items[0].status.podIP}')
+$ export INGRESS_POD_IP=$(kubectl -n istio-system get pod -l istio=ingress -o jsonpath='{.items[0].status.podIP}')
```

Comment:
ingress is going away in 1.0 - don't see a strong need to add it to documentation targeted at 1.0.

Reply:
Removed